bin/server_stop.sh

```diff
@@ -18,15 +18,17 @@ else
 fi
 
 pidFilePath=$appdir/$PIDFILE
+PID="$(cat "$pidFilePath")"
 
 if [ ! -f "$pidFilePath" ] || ! kill -0 "$(cat "$pidFilePath")"; then
    echo 'Job server not running'
```

Comment:
By my comment, I just noticed that in case Line 27 is not successful, a valid PID file may be possibly removed by my change. I think I need a flag variable `isValidPID` somewhere.